### PR TITLE
Add logback, so we get test outputs...

### PR DESCRIPTION
**Pull request by roesslerj**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -121,6 +121,13 @@
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>ch.qos.logback</groupId>
+			<artifactId>logback-classic</artifactId>
```
Comment (diba1013) [inline pom.xml:126]: You should replace slf4j-api with this. Why only tests for production it does not work also... .

File: src/test/resources/retest/logback.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration debugging="false">
```
Comment (diba1013) [inline src/test/resources/retest/logback.xml:2]: This is for retest for starting the gui.

**Review by diba1013 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (roesslerj) [reply pom.xml:126]: No. We deliver an API. It should be up to the user, to determine his logging framework. We use a concrete logging framework only in our own tests.

Comment (roesslerj) [reply pom.xml:126]: But maybe this should be put into the readme somewhere...

File: src/test/resources/retest/logback.xml (hunk 2 above)
Comment (roesslerj) [reply src/test/resources/retest/logback.xml:2]: Hmm. Does it hurt?

**Review by diba1013 — APPROVED** [review]

**Pull request merged**